src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,143 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node tail;
+    private Node head;
```

Comment:
Node should be parameterized
```suggestion
    private Node<T> tail;
    private Node<T> head;
```